unittests/scans/aws_prowler_v3plus/many_vuln.ocsf.json

```diff
@@ -67,7 +67,7 @@
         "provider": "aws",
         "region": "us-east-1"
     },
-    "event_time": "2024-06-03T14:15:19.382075",
+    "time_dt": "2024-06-03T14:15:19.382075",
```

Comment:
Instead of modifying existing tests that are presumably still valid for older versions of Prowler, would you mind adding a dummy scan file with the new `time_dt` key populated to ensure that both keys continue to work?

Reply:
@cneill smth like this?

Reply:
Yep, thanks! I notice that we're not actually checking the date for accuracy in the old unit tests, but if there are no failures at the parser level at least it ensures we're not hitting issues with trying to e.g. parse a `None` as a timestamp